src/features/game/events/sell.ts

```diff
@@ -26,6 +27,11 @@ type Options = {
 };
 export function sell({ state, action }: Options): GameState {
   const stateCopy = cloneDeep(state);
+  const bumpkin = stateCopy.bumpkin;
+
+  if (bumpkin === undefined) {
```

Comment:
@veenhdek We will need to make a new event `sellCrop` since this `sell` event is used in the current game. Since Bumpkins don't exist in the current game, this would throw an error for everyone 😢 

We will need to do API work for this, I would suggest removing this part of the PR and we can update it 👍

Reply:
We've just handled this case in a new event 😄 

https://github.com/sunflower-land/sunflower-land/pull/1598

Reply:
Removed! @adamhannigan